src/core/caniuse.js

```diff
@@ -14,21 +14,29 @@ export const name = "core/caniuse";
 const API_URL = "https://respec.org/caniuse/";
 
 export const BROWSERS = new Map([
-  ["and_chr", { name: "Android Chrome", path: "chrome" }],
-  ["and_ff", { name: "Android Firefox", path: "firefox" }],
-  ["and_uc", { name: "Android UC", path: "uc" }],
-  ["chrome", { name: "Chrome" }],
-  ["edge", { name: "Edge" }],
-  ["firefox", { name: "Firefox" }],
-  ["ios_saf", { name: "iOS Safari", path: "safari-ios" }],
+  ["and_chr", { name: "Android Chrome", path: "chrome", type: "mobile" }],
```

Comment:
Maybe create two maps? 
(Can merge when testing input for valid browsers)

Reply:
oh, you mean DESKTOP_BROWSERS and MOBILE_BROWSERS. I'll take a look.